.github/workflows/actionlint.yml

```diff
@@ -14,8 +18,10 @@ jobs:
   actionlint:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v3
+      - name: Checkout repository
+        uses: actions/checkout@v3
+
       - name: Check workflow files
         uses: docker://rhysd/actionlint:latest
         with:
-          args: -color
+          args: -ignore ".* was deprecated.*"
```

Comment:
Please open an issue to resolve these if we don’t already have one, and leave until comment above this suppression

Reply:
Done. There is an issue at #12696 already.